Workflows/Extensions/StringToInt.cs

```diff
@@ -0,0 +1,24 @@
+using Bonsai;
+using System;
+using System.ComponentModel;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+
+[Combinator]
+[Description("")]
+[WorkflowElementCategory(ElementCategory.Transform)]
+public class StringToInt
```

Comment:
Consider a more suggestive name and description. Looks like this is more of a lookup for a specified array of strings? Maybe `LookupStringIndex` and maybe if there is a defined meaning for these trings, e.g. if they are topics you could even make it more concrete like `LookupTopicIndex`.